### PR TITLE
Match ovl1_sf_tr360

### PR DESCRIPTION
**Pull request by Ryan-Myers**
Another small overlay file matched, with data migrated.

### CODE REVIEW

**Review by sonicdcer — APPROVED** [review]
Great quality of work, thank you for this!

File: src/overlays/ovl_i1/sf_tr360.c [hunk 1]
```diff
@@ -1,9 +1,233 @@
...
+    u16* msg;
+    u16 unk;
+} UnkMsg_D_i1_8019AE50;
+
+UnkMsg_D_i1_8019AE50 D_i1_8019AE50[] = {
+    { gMsg_ID_23028, 224 }, { gMsg_ID_23029, 160 }, { gMsg_ID_23030, 224 }, { gMsg_ID_23031, 256 },
+    { gMsg_ID_23016, 160 }, { gMsg_ID_23024, 160 }, { gMsg_ID_23019, 256 }, { gMsg_ID_23015, 160 },
+    { gMsg_ID_23017, 160 }, { gMsg_ID_23025, 160 }, { gMsg_ID_23002, 160 }, { gMsg_ID_23004, 160 },
+    { gMsg_ID_23005, 160 }, { gMsg_ID_23023, 160 }, { gMsg_ID_23006, 160 }, { gMsg_ID_23007, 256 },
+    { gMsg_ID_20330, 0 },   { gMsg_ID_23026, 256 }, { gMsg_ID_23032, 256 }, { NULL, 0xFFFF },
```
Comment (petrie911) [inline src/overlays/ovl_i1/sf_tr360.c:162]:
```suggestion
    { gMsg_ID_20330, 0 },   { gMsg_ID_23026, 256 }, { gMsg_ID_23032, 256 }, { NULL, -1 },
```
This looks better to me, even if it is technically an unsigned value.

File: src/overlays/ovl_i1/sf_tr360.c [hunk 2]
```diff
@@ -1,9 +1,233 @@
...
+bool func_i1_80198DCC(Actor* actor, f32 x, f32 z) {
+    Object_58* obj58;
+    s32 i;
+
+    for (i = 0, obj58 = gObjects58; i < 200;) {
+        if ((obj58->obj.status == 2) &&
+            (fabsf(obj58->obj.pos.x - (actor->obj.pos.x + ((x * 650.0f) + 350.0f))) < 1200.0f) &&
+            (fabsf(obj58->obj.pos.z - (actor->obj.pos.z + ((z * 650.0f) + 350.0f))) < 1200.0f) &&
+            (actor->obj.pos.y < 650.0f)) {
+            return TRUE;
```
Comment (petrie911) [inline src/overlays/ovl_i1/sf_tr360.c:41]:
```suggestion
            return true;
```
We prefer the lowercase macros, even though we do have both. Also I still can't believe this isn't a loop unroll.

**Comment by sonicdcer** [comment]
I'll fix those myself, merging

**Pull request merged**